numpyro/distributions/transforms.py

```diff
@@ -67,36 +68,39 @@ def _clipped_expit(x: ArrayLike) -> ArrayLike:
 class Transform(object):
     domain = constraints.real
     codomain = constraints.real
-    _inv = None
+    _inv: Optional[TransformT] = None
 
     def __init_subclass__(cls, **kwargs):
         super().__init_subclass__(**kwargs)
         register_pytree_node(cls, cls.tree_flatten, cls.tree_unflatten)
 
     @property
-    def inv(self) -> TransformT:
+    def inv(self: TransformT) -> TransformT:
         inv = None
         if self._inv is not None:
-            inv = self._inv()
+            inv = self._inv
```

Comment:
I think this might be causing the tests to fail ... the problem is that inv is a Transform and the associated protocol TransformT has a `__call__(x)` function that receives a parameter and this is why MyPy complains :/